bench/wtperf/config.c

```diff
@@ -652,6 +678,72 @@ config_sanity(CONFIG *cfg)
 	return (0);
 }
 
+void
+config_queue_dedupe(CONFIG *cfg)
```

Comment:
1.  The function should have a comment.
2.  Please call it `config_consolidate` as `dedupe` is not as descriptive.